lib/stylesheet/importer.rb

```diff
@@ -112,21 +115,11 @@ def import_color_definitions
       resolved_ids = Theme.transform_ids(theme_id)
 
       if resolved_ids
-        theme = Theme.find_by_id(theme_id)
-
-        contents << "\n\n// Theme SCSS variables\n\n"
-        contents << theme&.scss_variables.to_s.split(";").join(";\n") + ";\n\n"
-        contents << "\n\n"
```

Comment:
It doesn't look like we have a spec for `scss_variables` used in `color_definitions`. The refactor here looks safe (this has been moved to `ThemeField.compiled_css`, but it might be nice to add a spec for this.